src/scripts/GetChromeDriverVersion.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const request = require('sync-request');
+
+/*
+Getting chrome version driver from google
+*/
+const getChromeDriverVersion = () => {
```

Comment:
consider make  getChromeDriverVersion parametrised,
in case we'd like a chrome driver version for a specific chrome browser version.